mathesar_ui/src/components/abstract-type-control/AbstractTypeControl.svelte

```diff
@@ -92,34 +99,31 @@
 
   {#if selectedAbstractType && selectedDbType}
     {#key selectedAbstractType}
-      <AbstractTypeOptions
+      <AbstractTypeDBOptions
         {selectedAbstractType}
         bind:selectedDbType
         bind:typeOptions
-        bind:displayOptions
         {column}
       />
     {/key}
   {/if}
 
-  <div class="footer">
-    <CancelOrProceedButtonPair
-      onProceed={onSave}
-      onCancel={cancel}
-      isProcessing={typeChangeState?.state === 'processing'}
-      canProceed={!isSaveDisabled}
-      proceedButton={{ label: 'Save' }}
-      size="small"
-    />
-  </div>
+  {#if actionButtonsVisible}
+    <div class="footer">
+      <CancelOrProceedButtonPair
+        onProceed={onSave}
+        onCancel={cancel}
+        isProcessing={typeChangeState?.state === 'processing'}
+        canProceed={!isSaveDisabled}
+        proceedButton={{ label: 'Save' }}
+        size="small"
+      />
+    </div>
+  {/if}
```

Comment:
The `AbstractTypeControl` component does not contain the errors and info messages present in the figma design.

Reply:
0d8a9ae6
76eb99aa